config/bfo.yml

```diff
@@ -132,9 +135,12 @@ entries:
 - exact: /2014-05-03/bfo.owl
   replacement: https://github.com/BFO-ontology/BFO/raw/releases/2014-05-03/owl-group/bfo.owl
 
-- exact: /2015-8-29/bfo.owl
```

Comment:
This looks like it was a mistake, but I don't want to remove this PURL. I think we should keep `2015-8-29` and add `2015-08-29`.

Reply:
Actually this IRI do not implemented in the previous BFO OWL release file and used in anywhere. That's why I changed the link since it won't hurt anything. But it is a dated version IRI that pointed to first official release of BFO 2.0 (classes only) in case anyone would like to use in the future.